### PR TITLE
SS-GAN_TensorFlow

### PR DESCRIPTION
**Pull request by antreev-brar**
2nd Submission for model zoo

### CODE REVIEW

File: generative_models/SS-GAN_TensorFlow/utils.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+from numpy import asarray
+from numpy.random import randn
+from numpy.random import randint
+import keras
+from keras.layers import UpSampling2D ,Reshape, Conv2D , Dense ,Input , Lambda, BatchNormalization , LeakyReLU , Conv2DTranspose , Flatten , Dropout , Activation 
+from keras.models import Model
+from keras.datasets.mnist import load_data
+from keras import backend
+from keras.optimizers import Adam
+from matplotlib import pyplot
```
Comment (nirmal-suthar) [inline generative_models/SS-GAN_TensorFlow/utils.py:9]: use only those imports which are used in this file

File: generative_models/SS-GAN_TensorFlow/readme.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
+# Tensorflow Implementation of Semi Supervised GAN
+
+## Usage
+```bash
+$ python3 main.py
```
Comment (nirmal-suthar) [inline generative_models/SS-GAN_TensorFlow/readme.md:5]:
```suggestion
$ python3 main.py --epochs 20
```
It is better to include some arguments (for reference)

File: generative_models/SS-GAN_TensorFlow/model.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+from numpy import asarray
+from numpy.random import randn
+from numpy.random import randint
+import keras
+from keras.layers import UpSampling2D ,Reshape, Conv2D , Dense ,Input , Lambda, BatchNormalization , LeakyReLU , Conv2DTranspose , Flatten , Dropout , Activation 
+from keras.models import Model
+from keras.datasets.mnist import load_data
+from keras import backend
+from keras.optimizers import Adam
+from matplotlib import pyplot
```
Comment (nirmal-suthar) [inline generative_models/SS-GAN_TensorFlow/model.py]: use only those imports which are needed

File: generative_models/SS-GAN_TensorFlow/main.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+from keras.datasets.mnist import load_data
+from keras import backend
+from keras.optimizers import Adam
+from matplotlib import pyplot
+import argparse
+
+import tensorflow as tf
+import matplotlib.pyplot as plt
+from model import *
+from utils import *
```
Comment (nirmal-suthar) [inline generative_models/SS-GAN_TensorFlow/main.py]: use only those which are needed

**Comment by antreev-brar** [comment]
Nirmal , I have made the changes
